cds_ils/config.py

```diff
@@ -182,6 +183,9 @@ def _parse_env_bool(var_name, default=None):
     os.environ.get("HOSTNAME", ""),  # fix disallowed host error during /ping
 ]
 
+APP_DEFAULT_SECURE_HEADERS["content_security_policy"] = {}
```

Comment:
Please remove this line:
```suggestion
APP_DEFAULT_SECURE_HEADERS["content_security_policy"] = {}
```
We should actually remove it from ILS too, and set the right CSP headers!!!